index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"
+        integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;400&display=swap" rel="stylesheet">
+    <title>My portfolio</title>
+</head>
+
+<body>
+
+    <header>
+        <nav>
+            <div class="toolbar">
+                <a href="#" class="menu"><i class="fa-solid fa-bars"></i></a>
+                <h2>Mylogo</h2>
+            </div>
+        </nav>
+    </header>
+
+    <section class="headline">
+
+        <h1 id="Headline-Title">I’m Christelle,
+            Glad to see <br> you!</h1>
+        <p id="Headline-Text">I’m a software developer! I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate
+            to contact me. </p>
+        <div class="socialmedia">
+            <h3 class="helpingtext">LET'S CONNECT</h3>
+            <div class="belowtext">
+                <ul>
+                    <li>
+                        <a href="#"><img src="req/twitter.png" alt="twitter"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="req/Linkedin.png" alt="linkedin"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="req/M.png" alt="M"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="req/Github.png" alt="Github"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="req/V.png" alt="V"></a>
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </section>
+</body>
+
+</html>
```

Comment:
Kindly work on the responsiveness of your project, presently it is not responsive. It is just one side on the mobile view of 740px 😞 

<img width="717" alt="image" src="https://user-images.githubusercontent.com/73854999/219873219-82b7fd0b-cbe6-45a2-8421-6aa6a7c557d0.png">